Microsoft.Azure.CosmosRepository/src/Providers/ICosmosThroughputProvider.cs

```diff
@@ -1,20 +1,23 @@
 ﻿// Copyright (c) IEvangelist. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using Microsoft.Azure.Cosmos;
 
 namespace Microsoft.Azure.CosmosRepository.Providers
 {
     /// <summary>
     /// Exposes the ability to get an <see cref="IItem"/>s containers throughput properties.
     /// </summary>
-    public interface ICosmosThroughputProvider
+    interface ICosmosThroughputProvider
```

Comment:
Is this a breaking change?

Reply:
Potentially yes, these should not really be external however, they seem to have slipped through the net, what is your thought's on this what should we do?

Reply:
I'm fine with it, probably wasn't used externally.

Reply:
Okay cool we'll leave it as is 😃